framework/src/main/scala/org/sunbird/obsrv/core/model/SystemConfig.scala

```diff
@@ -1,13 +1,16 @@
 package org.sunbird.obsrv.core.model
 
+import org.sunbird.obsrv.core.util.SystemConfigSelector
+import org.sunbird.obsrv.core.model.Models.SystemSettings
 object SystemConfig {
+  def getSystemConfig(key: String): Option[SystemSettings] = {
+    SystemConfigSelector.getSystemConfigurations().find(config => config.objectKey == key)
+  }
 
-  // TODO: Fetch the system config from postgres db
-  val defaultDedupPeriodInSeconds: Int = 604800 // 7 days
-  val maxEventSize: Long = 1048576
-  val defaultDatasetId = "ALL"
-
-  // secret key length should be 16, 24 or 32 characters
-  val encryptionSecretKey = "ckW5GFkTtMDNGEr5k67YpQMEBJNX3x2f"
+  val defaultDedupPeriodInSeconds: Int = getSystemConfig("defaultDedupPeriodInSeconds").map(_.intValue()).getOrElse(604800) // 7 days
+  val maxEventSize: Long = getSystemConfig("maxEventSize").map(_.longValue()).getOrElse(1048576) // 1 MB
+  val defaultDatasetId: String = getSystemConfig("defaultDatasetId").map(_.stringValue()).getOrElse("ALL")
 
+  // secret key length should be 32 characters
+  val encryptionSecretKey: String = getSystemConfig("encryptionSecretKey").map(_.stringValue()).getOrElse("ckW5GFkTtMDNGEr5k67YpQMEBJNX3x2f")
```

Comment:
No default encryption key in the code. Let there be default during the installation